AF-2038 - webhook cert rotation, update wording (#118)
PagerDuty/developer-docs: docs/webhooks/08-Certificates.md

PagerDuty Webhooks provide client certificates when requested by a server (Mutual TLS).  

> It is our recommendation that customers configure their servers to keep an up-to-date certificate bundle that includes DigiCert root certificates in their local trust store. Customers choosing to rely on the PagerDuty client certificate are responsible for rotating to the new certificates at the appropriate time in order to avoid interrupted connectivity. PagerDuty webhook client certificates are rotated yearly. The current certificates are provided below.

Reviewer: Just omitted extra words for better readability.
```suggestion
We recommend that customers configure their servers to keep an up-to-date certificate bundle that includes DigiCert root certificates in their local trust store. Customers relying on the PagerDuty client certificate are responsible for rotating to the new certificates at the appropriate time to avoid interrupted connectivity. PagerDuty webhook client certificates are rotated yearly. The current certificates are provided below.
```

#### Client Certificates
Certificate | Common Name | Start Date | End Date | Download
---------|----------|---------|---------|---------

>  US region certificate | webhooks.pagerduty.com | December 18th, 2024 | December 2025 | [link](https://developer.pagerduty.com/certificates/2025_webhooks_pagerduty_com.pem)

Reviewer 2: Do we need a specific end date, like we had before for the "current" ones?

If not, I wonder if we should say November instead. If the last certificate's end date was 17 days shy of a year, that puts this one's end date at December 1st.

Reviewer: +1 on this
